website/docs/sdk-reference/unreal.md

```diff
@@ -0,0 +1,712 @@
+
+| Properties              | Description                                                                                                                                                                                                                                                                                         |
+| ----------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `SdkKey`                | SDK Key to access your feature flags and configurations. Get it from ConfigCat Dashboard.                                                                                                                                                                                                           |
+| `BaseUrl`               | Optional, sets the CDN base url (forward proxy, dedicated subscription) from where the sdk will download the config JSON.                                                                                                                                                                        |
+| `DataGovernance`        | Optional, defaults to `Global`. Describes the location of your feature flag and setting data within the ConfigCat CDN. This parameter needs to be in sync with your Data Governance preferences. [More about Data Governance](advanced/data-governance.md). Available options: `Global`, `EuOnly`.  |
+| `ConnectTimeoutMs`      | Optional, defaults to `8000ms`. Sets the amount of milliseconds to wait for the server to make the initial connection (i.e. completing the TCP connection handshake). `0` means it never times out during transfer                                                                                  |
+| `ReadTimeoutMs`         | Optional, defaults to `5000ms`. Sets the amount of milliseconds to wait for the server to respond before giving up. `0` means it never times out during transfer.                                                                                                                                   |
+| `PollingMode`           | Optional, sets the polling mode for the client. [More about polling modes](#polling-modes).                                                                                                                                                                                                         |
+| `AutoPollInterval`      | For PollingMode == Custom, sets at least how often this policy should fetch the latest configuration and refresh the cache.                                                                                                                                                                         |
```

Comment:
```suggestion
| `AutoPollInterval`      | For PollingMode == Custom, sets at least how often this policy should fetch the latest config JSON and refresh the cache.                                                                                                                                                                         |
```

Reply:
I missed this suggestion, I will apply the change manually next commit.